catalog/ui/catalog-ui-search/src/main/java/org/codice/ddf/catalog/ui/util/EndpointUtil.java

```diff
@@ -181,6 +192,29 @@ public Metacard getMetacard(String id)
             }
             resultTypes.put(metacardType.getName(), attributes);
         }
+        for (InjectableAttribute attribute : injectableAttributes) {
```

Comment:
probably not because we're iterating over different objects  (two different interfaces).